tests/translate/misc/test_quote.py

```diff
@@ -149,6 +149,11 @@ def test_propertiesdecode():
         assert quote.propertiesdecode("abc\\") == "abc\\"
         assert quote.propertiesdecode("abc\\") == "abc\\"
 
+    @staticmethod
+    def test_controlchars_round_trip():
```

Comment:
It would be good to also check that the plain encode & decode operations work as expected, i.e. that:
```python
encode('\x01') == '\\u0001'
decode('\x01') == '\x01'
decode('\\u0001') == '\x01'
```